source/reference/compatibility.txt

```diff
@@ -202,6 +202,37 @@ in ``mongosh`` better align with the types used by the MongoDB Drivers.
    For more information on managing types, refer to the
    :manual:`schema validation overview </core/schema-validation>`.
 
+Undefined Values
+----------------
+
+The undefined BSON type is `deprecated <https://bsonspec.org/spec.html>`__ in
+:binary:`bin.mongosh`. If you insert a document with the undefined BSON type in
+:binary:`bin.mongosh`, your deployment stores a null value. To create undefined
+fields in your documents, use the legacy :binary:`~bin.mongo` shell.
```

Comment:
(suggestion)
Maybe omit this last sentence because I don't think we want to point folks to the legacy shell. Is there maybe another solution?

Reply:
After a bit of investigation, it seems that generally users should never actually need to insert an undefined value, so I'm thinking we might be okay to not provide a work around? I will get more opinions on this though. It seems if we really want to point them at a workaround, they could use a driver that supports inserting raw BSON data, e.g. through a Python script